.github/workflows/examples-efr32.yaml

```diff
@@ -88,25 +88,25 @@ jobs:
           ./scripts/run_in_build_env.sh \
              "./scripts/build/build_examples.py \
                 --enable-flashbundle \
-                --target efr32-brd4187c-window-covering \
-                --target efr32-brd4187c-switch \
-                --target efr32-brd4187c-switch-sed \
+                --target efr32-brd4187c-window-covering-additional_data_advertising \
+                --target efr32-brd4187c-switch-openthread_mtd-shell \
+                --target efr32-brd4187c-switch-sed-low-power-use_ot_coap_lib \
```

Comment:
Why are there unrelated workflow changes here?

Reply:
interesting how this crept in. Maybe during rebase. Fixed

Reply:
Now I'm seeing unrelated changes in the opposite direction, removing changes that have landed....  This PR should just not be changing this file!